### ISSUE TITLE
fileio_io.py

### ISSUE THREAD

**Issue by yaoguangshun** [open-mmlab/mmcv#798]
```python
import mmcv
# load data from a file-like object
with open('test.json', 'r') as f:
    data = mmcv.load(f)
```
can not load data from a file-like object? It's some bug here?

**Comment by ycxioooong** [comment]
Hi @yaoguangshun, thanks for reporting this issue. 
I find that the instruction in https://github.com/open-mmlab/mmcv/blob/master/docs/io.md is misleading and need some updates.

If you want to use `mmcv.load` with Python Context Manager, you need to parse a `file_format` parameter to `mmcv.load` :
```python
import mmcv
# load data from a file-like object
with open('test.json', 'r') as f:
    data = mmcv.load(f, file_format='json')
```

**Comment by yaoguangshun** [comment]
OK, thanks for replying，the instruction in https://github.com/open-mmlab/mmcv/blob/master/docs/io.md is old and  need some updates

**Comment by ycxioooong** [comment]
Fixed by #813
